code/src/components/Popular.js

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from "react"
+import { Link } from "react-router-dom"
+import styled from "styled-components"
+
+import { MOVIES_URL } from 'utils/urls'
+
+
+const Popular = () => {
+
+    const [movieList, setMovieList] = useState([])
+
+    //moved fetch here since we want the fetch to happen only when this Route (/) opens and not for every visit on our website.
+
+    useEffect(() => {
+        fetch(MOVIES_URL)
+            .then(res => res.json())
+            .then(data => {
+                setMovieList(data.results)
+            })
+    }, [])
+
+    return (
+        <Main>
+            {movieList.map((movie => {
+                return (
+                    <Link to={`/movies/${movie.id}`} key={movie.id}>
+                        <FilmCard>
+                            <FilmImageGroup>
+                                <MovieDetails>
+                                    <MovieCard src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt={movie.title} />
+                                    <ImageOverlay></ImageOverlay>
+                                    <TextOverlay>
+                                        <Filmtitle>{movie.title}</Filmtitle>
+                                        <p>Released {movie.release_date}</p>
+                                    </TextOverlay>
+                                </MovieDetails>
+                            </FilmImageGroup>
+                        </FilmCard>
+                    </Link>
+                )
+            }))}
+        </Main>
+    )
+}
+
+export default Popular
+
+//styled components
+const Main = styled.main`
+display: grid;
+grid-template-columns: 1fr;
+
+
+@media (min-width: 768px) {
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+@media (min-width: 1100px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
One suggestion for the media queries is if you want a little smoother transition, you could go to the second size on a lower pixel amount and go to two images instead of three. 
/ Ida & Simon